portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +28,56 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+    private List<String> quotes;
+    private List<String> jstrings;
+
+  @Override
+  public void init() {
+    quotes = new ArrayList<>();
+    quotes.add("Si se Puede");
+    quotes.add("Shout out to my mom");
+    quotes.add("Siempre Humble");
+    quotes.add("Adelante y con ganas");
+    quotes.add("Don't complain you have too much on your plate when your goal was to eat");
+    quotes.add("She believes she could so she did");
+
+    jstrings = new ArrayList<>();
+    jstrings.add("I hope this string wroks");
+    jstrings.add("And also this string");
+    jstrings.add("This the third string");
+  }
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
The purpose of this method is to return an array of strings. If all of your code was written in Java, then you could just return a Java array. However, your frontend code is written in Javascript, so you can't just return a Java code.

That's what JSON is for; a Java object is just some data in memory and JSON converts that data in memory to a human-readable string. Then Javascript parses that human-readable string into its own memory representation of that object.

So the overall flow looks like:
The web server stores an array in memory -> The web server converts the array in memory to a JSON string -> The web server sends the JSON string to Javascript in the client -> The Javascript in the client converts that JSON string into an object -> The Javascript client stores the object in memory.

Does that make sense?